python/dgl/distributed/graph_services.py

```diff
@@ -146,7 +156,7 @@ def _sample_neighbors_graphbolt(
 
     return_eids = g.edge_attributes is not None and EID in g.edge_attributes
     subgraph = g._sample_neighbors(
-        nodes, None, fanout, replace=replace, return_eids=return_eids
+        nodes, seed_offsets, fanout, replace=replace, return_eids=return_eids
```

Comment:
Are `nodes` required to be sorted in node types no matter `seed_offsets` is None or not?